lib/xcodeproj/plist.rb

```diff
@@ -1,10 +1,10 @@
+autoload :Nanaimo, 'nanaimo'
+autoload :CFPropertyList, 'cfpropertylist'
```

Comment:
I don't like requiring everything upfront -- that would mean just `require`ing Xcodeproj would cause a large amount of potentially irrelevant code to be loaded upfront